packages/smart-contracts/src/lib/artifacts/BatchConversionPayments/index.ts

```diff
@@ -13,52 +13,48 @@ export const batchConversionPaymentsArtifact = new ContractArtifact<BatchConvers
           address: '0x2e335F247E91caa168c64b63104C4475b2af3942',
           creationBlockNumber: 0,
         },
-        rinkeby: {
-          address: '0x42c466ccF141A8EBE0Aa2ECEF06A4f4BE9a9B526',
-          creationBlockNumber: 11338182,
-        },
         goerli: {
-          address: '0x42c466ccF141A8EBE0Aa2ECEF06A4f4BE9a9B526',
-          creationBlockNumber: 7546738,
-        },
-        mainnet: {
-          address: '0x42c466ccF141A8EBE0Aa2ECEF06A4f4BE9a9B526',
-          creationBlockNumber: 15537996,
+          address: '0x0fD49d410fB50Dc1f308975E5D19B5c4551B0041',
+          creationBlockNumber: 7791166,
         },
+        // mainnet: {
```

Comment:
can't merge until it's deployed.

Reply:
Sure, I am trying to get every approval, for both queries. And in the meantime, to have more ETH on the admin address to deploy it before to update once for all